pages/devs/usecases.mdx

```diff
@@ -2,28 +2,33 @@
 
 To provide you with inspiration, we have highlighted several integration examples below.
 
-**Domain & PFP after connecting wallet *([Carbonable](https://app.carbonable.io))***
+**Domain & PFP after Connecting Wallet *([Carbonable](https://app.carbonable.io))***
 
 ![img](public/illustrations/carbonable-integration.webp)
 
-**Domain & PFP on wallet description *([Bountive](https://app.bountive.fi/))***
+**Domain & PFP on Wallet Description *([Bountive](https://app.bountive.fi/))***
 
 ![img](public/illustrations/bountive-integration.webp)
 
-**Domain & PFP on leaderboard *([Starknet Quest](https://starknet.quest/))***
+**Domain & PFP on Leaderboard *([Starknet Quest](https://starknet.quest/))***
 
 ![img](public/illustrations/sq-integration.webp)
 
-**Research integration *([Argent](https://portfolio.argent.xyz/))***
+**User Research *([Argent](https://portfolio.argent.xyz/))***
 
 ![img](public/illustrations/argent-integration.webp)
 
-**Braavos subdomain *([Braavos](https://braavos.app/))***
+**Send Money Through Domains *([Argent](https://www.argent.xyz/argent-x/))***
 
-![img](public/illustrations/braavos-integration.webp)
+![img](public/illustrations/send.webp)
+
+**Get your [Starknet ID](http://Starknet.id) using Affiliation *([Flex](https://hyperflex.market/))***
+
+![img](public/illustrations/flex.webp)
 
-**Get your [Starknet ID](http://Starknet.id) using affiliation *([Flex](https://hyperflex.market/))***
+**Register A Subdomain *([Braavos](https://braavos.app/))***
```

Comment:
Here you're not showing the registration page of a subdomain but it's result (that could be seen on argent also). 

So we want you to showcase the page to register subdomains imo